testsuite/integration/elytron/pom.xml

```diff
@@ -903,6 +905,70 @@
+                </property>
+            </activation>
+            <properties>
+                <maven.local.repo>${wildfly.transformed.repo.dir}</maven.local.repo>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.wildfly.plugins</groupId>
+                        <artifactId>wildfly-maven-plugin</artifactId>
+                        <version>${version.org.wildfly.plugin}</version>
+                        <configuration>
+                            <system-properties>
+                                <maven.repo.local>${wildfly.transformed.repo.dir}</maven.repo.local>
+                            </system-properties>
+                        </configuration>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.jboss.galleon</groupId>
+                        <artifactId>galleon-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>server-provisioning</id>
+                                <goals>
+                                    <goal>provision</goal>
+                                </goals>
+                                <phase>generate-resources</phase>
+                                <configuration>
+                                    <plugin-options combine.self="append">
+                                        <jboss-jakarta-transform-artifacts>false</jboss-jakarta-transform-artifacts>
+                                        <jboss-maven-provisioning-repo>${maven.local.repo}</jboss-maven-provisioning-repo>
+                                    </plugin-options>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <executions>
+                            <!--Re-enable the default surefire execution. -->
+                            <execution>
+                                <id>default-test</id>
+                                <phase>test</phase>
+                                <configuration>
+                                    <systemPropertyVariables>
+                                        <server.jvm.args>-Dmaven.repo.local=${wildfly.transformed.repo.dir}</server.jvm.args>
+                                    </systemPropertyVariables>
+                                    <!-- These include a module to be used which currently uses javax and is not transformed -->
+                                    <excludedGroups>org.jboss.as.test.shared.categories.RequiresTransformedClass</excludedGroups>
+                                </configuration>
+                            </execution>
```

Comment:
This is the change I forgot to push for some reason. The `UndertowSSLv2HelloTestCase` will pass by adding:
```xml
<execution>
    <id>sslv2-tests</id>
    <phase>test</phase>
    <configuration>
        <systemPropertyVariables>
            <server.jvm.args>-Dmaven.repo.local=${wildfly.transformed.repo.dir} -Djava.security.properties=${security.properties}</server.jvm.args>
        </systemPropertyVariables>
        <!-- These include a module to be used which currently uses javax and is not transformed -->
        <excludedGroups>org.jboss.as.test.shared.categories.RequiresTransformedClass</excludedGroups>
    </configuration>
</execution>
```

Reply:
Good catch, @jamezp!